### ISSUE TITLE
Feature Suggestion: Stable and Development Releases

### ISSUE THREAD

**Issue by dot-bob** [MiSTer-devel/Main_MiSTer#457]
With the MiSTer community starting to become larger and more diverse with people of varying technical ability it might be helpful to implement Development and Stable Releases.  Bleeding edge binaries are contained in the Development Release and occasionally when the development is deemed stable a snapshot is made and moved to the Stable Release.

To facilitate this release method two update scripts could be used.  The current script "update" could be used as the stable release and a second script named like "update_experimental", "update_dev" , "update_beta" or something to that effect.  Those unaware of the changes or want to be on stable update as they always have and those who want to be on the bleeding edge use the new script.

To switch between releases one would just start the opposite script and it would perform a full update of the images taking the installed binaries from stable to development or vice versa.

This could also help users determine if a recent release broke something or it is just their setup.

**Comment by sorgelig** [comment]
It's already done.
There are unofficial nightly builds considered unstable.
Formal releases are considered stable but there are no dedicated testers doing comprehensive tests, so bugs are possible.

But overall this project is hobby and it's developed as is without obligations. Many people provides patches and sometimes a whole cores but all of it is hobby. And it's inappropriate to apply restrictions hard to follow.
Take it as is.
